content/source/docs/enterprise/api/ptfe_settings.html.md

```diff
@@ -0,0 +1,491 @@
+---
+layout: enterprise2
+page_title: "Private Terraform Settings - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-private-terraform-settings"
+---
+
+# Private Terraform Settings API
+
+-> **Note**: These API endpoints are in beta and are subject to change. Additionally, these endpoints are only available in Private Terraform instances and only accessible by site administrators.
+
+## List General Settings
+`GET /api/v2/admin/settings/general`
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) which has the `site-admins` permission.
+
+Status  | Response                                         | Reason
+--------|--------------------------------------------------|-------
+[200][] | Array of [JSON API document][]s (`type: "settings"`) | Successfully listed General settings
```

Comment:
From the examples below, I don't think this is accurate—it doesn't look like an array, just a single document? That would make sense, since we're semantically fetching a single record.

(Same comment throughout)

Reply:
Copy pasta error from template. Good catch, will resolve!